packages/circle-demo-webapp/app/routes/wallets.$id/components/EditWalletSetDialog.tsx

```diff
@@ -10,12 +10,12 @@ import {
   DialogTitle,
   DialogTrigger,
 } from '~/components/ui/dialog';
-import { WalletSet } from '~/lib/types';
+import { ElementsWalletSet } from '~/lib/types';
 
 import { EditWalletSetForm } from './EditWalletSetForm';
 
 interface EditWalletSetDialogProps {
-  walletSet: WalletSet;
+  walletSet: ElementsWalletSet;
```

Comment:
@avkos I renamed our types (that extend Circle types) to prefix them with `Elements*` to make it obvious that these are react-elements types.